src/lib/FreeC/Pass/ExportPass.hs

```diff
@@ -62,6 +66,9 @@ where
 
 import qualified Data.Map.Strict               as Map
 import qualified Data.Set                      as Set
+import qualified Data.Text                     as Text
+
+import           Language.Coq.Gallina
```

Comment:
Use the `FreeC.Backend.Coq.Syntax` wrapper module instead of `language-coq` directly.

```suggestion
import qualified FreeC.Backend.Coq.Syntax      as Coq
```